nginx/configuration_builder/templates/nginx_location_template.conf.j2

```diff
@@ -55,4 +55,10 @@
         {% for header in headers %}
         proxy_set_header        {{ header }} $turnpike_{{ header.lower().replace("-", "_") }};
         {% endfor %}
+
+        # timeouts
+        send_timeout {{ timeout }};
+        client_body_timeout {{ timeout }};
+        proxy_read_timeout {{ timeout }};
+        proxy_send_timeout {{ timeout }};
```

Comment:
- [ ] add the following config as well: 
```
proxy_request_buffering off;
proxy_buffering         off;
proxy_max_temp_file_size 0;
```

but make the values also configurable per route